### ISSUE TITLE
version 0.6.2 broken?

### ISSUE THREAD

**Issue by theyosh** [pywemo/pywemo#248]
After updating to version 0.6.2 I get the following error importing the pywemo module:

```python
>>> import pywemo
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/dist-packages/pywemo/__init__.py", line 4, in <module>
    from .discovery import discover_devices, setup_url_for_address
  File "/usr/local/lib/python3.7/dist-packages/pywemo/discovery.py", line 9, in <module>
    from . import ssdp
  File "/usr/local/lib/python3.7/dist-packages/pywemo/ssdp.py", line 12, in <module>
    from .ouimeaux_device.api.long_press import VIRTUAL_DEVICE_UDN
  File "/usr/local/lib/python3.7/dist-packages/pywemo/ouimeaux_device/__init__.py", line 21, in <module>
    from .api.service import REQUESTS_TIMEOUT, Service, Session
  File "/usr/local/lib/python3.7/dist-packages/pywemo/ouimeaux_device/api/service.py", line 28, in <module>
    class Session:
  File "/usr/local/lib/python3.7/dist-packages/pywemo/ouimeaux_device/api/service.py", line 57, in Session
    total=6, backoff_factor=1.5, allowed_methods=['GET', 'POST']
TypeError: __init__() got an unexpected keyword argument 'allowed_methods'
```
With version 0.6.1 there is no problem.

Did I miss something?

**Comment by esev** [comment]
It looks like `allowed_methods` wasn't added until urllib3 version 1.26.0. Updating to that version of urllib should fix the issue. 
https://github.com/urllib3/urllib3/blob/1.26.0/src/urllib3/util/retry.py

I'll send a PR to fix the pywemo dependencies.

**Comment by theyosh** [comment]
Ah, that did the trick. Upgrading urllib3 to the latest version and then pywemo does also work at the latest version.
Thanks for the info.

**Comment by esev** [comment]
@theyosh thanks for verifying!

**Comment by esev** [comment]
Keeping this open till the next version bump.

**Comment by pavoni** [comment]
Thanks for catching this - now released in `0.6.3`
